### PR TITLE
Workshop 1.5

### PR DESCRIPTION
**Pull request by jenniferstruthers1-nhs**
Workshop 1.5 Covers:

- Recap of first workshop using py file
- if/elif/else statements
- loops
- homework tasks

### CODE REVIEW

File: workshop_2/loops.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+
+# While:
+# A While loop will repeat itself until a certain condition is satisfied
+# This can make them tricky, because if you're not careful they will never end!
+# To cancel, use ctr+c
+"""
+count = 0
+while count <= 5:
+    count = count #+ 1
```
Comment (warren-davies4) [inline workshop_2/loops.py:12]: is the hash mean to be there?

File: workshop_2/loops.py [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+while 1 + 1 == 2:
+    count = count + 1
+    print("count:", count)
+    if count == 10:
+        print("STOP!")
+        break
+"""
+
+# For:
+# For loops are much for commonly used
```
Comment (warren-davies4) [inline workshop_2/loops.py]:
```suggestion
# For loops are much more commonly used
```

File: workshop_2/loops.py [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+# They are used for doing things a certain number of times
```
Comment (warren-davies4) [inline workshop_2/loops.py]: used to loop through an iterable

File: workshop_2/homework.py [hunk 4]
```diff
@@ -0,0 +1,235 @@
...
+# This is some homework to complete before the next session.
+# Uncomment the code and replace the ???? with the correct piece of code to obtain the expected output
+# Run it after you complete each exercise to make sure you got it correct
+
+
+# Conditionals
+"""
+# Expected Output: Good morning
+time_of_day = "morning"
+if ????:
```
Comment (warren-davies4) [inline workshop_2/homework.py:11]: this is a good idea!

**Review by warren-davies4 — APPROVED** [review]
This is great! We can definitely use this for future engagements. Couple of minor suggestions.

File: workshop_2/loops.py (hunk 1 above)
Comment (jenniferstruthers1-nhs) [reply workshop_2/loops.py:12]: yeah, so I can show what happens without it

**Pull request merged**